README.md

```diff
@@ -75,20 +77,13 @@ existing proposals.
 [16](https://github.com/aws/aws-cdk-rfcs/issues/16)|[RFC Process](https://github.com/aws/aws-cdk-rfcs/pull/53)|[@MrArnoldPalmer](https://github.com/MrArnoldPalmer)|resolved
 <!--END_TABLE-->
 
-## What does all this mean?
-
-This document is a lot of information about process thats meant to help guide.
-It is not a set of rules that need to be strictly applied. It is designed to
-help contributors (and thats you!) become more involved with the tools that they
-rely on. All efforts to contribute are encouraged and appreciated.
-
 ## What is an RFC?
 
 An RFC is a document that proposes and details a change or addition to the CDK,
-JSII, and other related tooling. It also is a process for reviewing and
+jsii, and other related tooling. It also is a process for reviewing and
```

Comment:
The only nitpick I have is all of our other acronyms we capitalize, ie "CDK", "AWS" etc so for consistency I'd argue it should be "JSII" though obviously not something important.

Reply:
We've been using `jsii` lowercase across the board. If @RomainMuller and you prefer that we change that I think we should probably change it everywhere and not just here.